osl/preprocessing/batch.py

```diff
@@ -259,7 +260,15 @@ def load_config(config):
 
     return config
 
+def append_preprocinfo(dataset, config):
+    preprocinfo = f"\n\nOSL BATCH PROCESSING APPLIED ON {datetime.datetime.today().strftime('%d/%m/%Y %H:%M:%S')} \n%% config start %% \n{config} \n%% config end %%")
+    dataset['raw'].info['description'] = dataset['raw'].info['description'] + preprocinfo
+ 
+    if dataset['epochs'] is not None:
+        dataset['raw'].info['description'] = dataset['raw'].info['description']
```

Comment:
Should this be:

`dataset['epochs'].info['description'] = dataset['epochs'].info['description'] + preprocinfo`

Not sure why its assigning to 'raw' here?

Reply:
Totally! My bad!